etstool.py

```diff
@@ -123,6 +123,26 @@ def install(runtime, environment):
     click.echo('Done install')
 
 
+@cli.command(name='install-docs')
+@click.option('--runtime', default='3.6')
+@click.option('--environment', default=None)
+def install_docs(runtime, environment):
+    """ Install project and dependencies into a clean EDM environment, then
+    install further dependencies required for building documentation.
+
+    """
+    parameters = get_parameters(runtime, environment)
+    args = ['--runtime={}'.format(runtime)]
+    install(args=args, standalone_mode=False)
```

Comment:
It's a bit awkward to be calling one click function from another, here, and it's not clear to me that the call is correct: are we passing the `--environment` argument on?

I'd suggest refactoring this: we could move the common functionality into a private helper function that's called by both `install` and `install_docs`, or we could turn the `install_docs` functionality into an option (`with_docs`?) to the `install` command.

Reply:
Alternatively, we could keep things simple and just have one function: have `install` install the Sphinx theme as part of its work. It's already installing Sphinx and its dependencies.

Reply:
The set-up now is: `install` installs the documentation theme unless you pass a flag requesting it not do that. Arguably it could also not install sphinx if you pass the flag, but I don't want to mess around with the existing script too much!